extensions/ql-vscode/src/remote-queries/export-results.ts

```diff
@@ -100,6 +100,18 @@ async function exportResultsToGist(
   }
 }
 
+/**
+ * Builds Gist description
+ * Ex: Empty Block (Go) x results (y repositories)
+ */
+export const buildGistDescription = (query: RemoteQuery, analysesResults: AnalysisResults[]) => {
+  const resultCount = sumAnalysesResults(analysesResults);
+  return `${query.queryName} (${query.language}) ${resultCount} results (${query.numRepositoriesQueried} repositories)`;
```

Comment:
1. I didn't think of this earlier: for remote queries that were triggered before https://github.com/github/vscode-codeql/pull/1427, `numRepositoriesQueried` will be undefined 😿 
   ```
   Empty Block (javascript) 700 results (undefined repositories)
   ```
   Perhaps we should omit the repositories bit if the number is undefined? 

2. More minor, but we should make "repositories" singular if `numRepositoriesQueried` is 1 🤪 We can probably copy some of the logic from here:

   https://github.com/github/vscode-codeql/blob/f992679e943e147dadd1cb3781765c4c5a8954a0/extensions/ql-vscode/src/history-item-label-provider.ts#L69-L75

Reply:
Thanks Shati, you're the best! Have added the two changes.